doc/trash.md

```diff
@@ -29,11 +29,28 @@ the collection of items currently in the trash, simply iterate over it.
 ```java
```

Comment:
I've checked that java sample is not present on https://developer.box.com/reference/get-folders-trash-items/.
This is probably because of invalid sample tag. I think it should be `get_folders_trash_items` instead of `get_trash`